### DISCUSSION TITLE
Is there a setting that makes the decompiled code cleaner?

### DISCUSSION THREAD

**Discussion by ThunderCls** [dnSpyEx/dnSpy#172] [Q&A] [answered]
dnSpyEx version: 6.3.0 x64
.NET version: 6

dnSpyEx
![image](https://user-images.githubusercontent.com/13303069/220145768-266a5c39-5931-478b-ac03-9b239fa35a20.png)

dotPeek
![image](https://user-images.githubusercontent.com/13303069/220145918-483c8ca0-2af9-4b30-be06-ba4631a89219.png)

You can clearly see the difference between the code provided by dnSpyEx and dotPeek. Is there any setting or checkbox inside dnSpyEx that I'd need to select/deselect to get a cleaner code like the one provided by dotPeek? Or this is just how the decompiler engine works?

**Comment by GazziFX** [comment, marked as answer]
DnSpy based on older ILSpy decompiler engine https://github.com/dnSpyEx/dnSpy/issues/5
